spring-session/src/main/java/org/springframework/session/web/http/OnCommittedResponseWrapper.java

```diff
@@ -529,6 +530,14 @@ public void println(double d) throws IOException {
 			this.delegate.println(d);
 		}
 
+		public boolean isReady() {
```

Comment:
since this is not in the couchbase package, probably needs to be double-checked by @rwinch